portfolio/src/main/java/com/google/sps/servlets/NewServlet.java

```diff
@@ -1,19 +1,32 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
 
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
 
-@WebServlet("/servlet")
-public class NewServlet extends HttpServlet {
 
+@WebServlet("/step_one_servlet")
```

Comment:
Try a more descriptive name! Also, I think your other PR already had this servlet -- are you re-writing it again? If this branch depends on that branch, it might be a good idea to have a branch off a branch. Take a look at this, it might prove helpful: https://www.atlassian.com/git/tutorials/using-branches